### DISCUSSION TITLE
Upgrade: Cloudstack Management Upgrade

### DISCUSSION THREAD

**Discussion by kenneth0595** [apache/cloudstack#9938] [Q&A] [answered]
ISSUE TYPE
Cann't Upgrade Cloudstak

VERSION
From Cloudstack version 4.19.1.2 To 4.19.1.3

HYPERVISOR
CentOS 7.9 KVM

REPO
[apache-cloudstack]
name=Apache CloudStack
baseurl=http://download.cloudstack.org/centos/$releasever/4.19/
enabled=1
gpgcheck=0

UPGRADE OUTPUT
yum upgrade cloudstack-management

Loaded plugins: fastestmirror, langpacks
Loading mirror speeds from cached hostfile
 * epel: d2lzkl7pfhq30w.cloudfront.net
No packages marked for update

**Comment by kenneth0595** [comment]
@Wei Zhou
I've changed baseurl to https://download.cloudstack.org/centos7/4.19/ but it was the same issue that I could see.
This repo doesn't have 4.19.1.3 version.

**Reply by DaanHoogland** [reply]
@kenneth0595 4.19 no longer supports el7 distros. see https://docs.cloudstack.apache.org/en/latest/releasenotes/compat.html

**Reply by kenneth0595** [reply]
@DaanHoogland 
Thanks you for your quick response.

I will migrate it.

**Reply by weizhouapache** [reply]
actually el7 is still supported by ACS 4.19
we could build 4.19.1.3 packages for centos7  (we have only 4.19.1.2 packages now)

**Reply by DaanHoogland** [reply]
I noticed this, I really thought we had already deprecated el7. I am building the el7 as we chat.

**Reply by kenneth0595** [reply]
I really appreciate it.

@weizhouapache 
Is there a plan to support upgrading acs 4.20 on CentOS7 KVM?

**Reply by weizhouapache** [reply]
> I really appreciate it.
> 
> @weizhouapache 
> Is there a plan to support upgrading acs 4.20 on CentOS7 KVM?

Centos7 is no longer supported, It is EOL.
java 11 is EOL as well.

You need to upgrade to RHEL 8/9 or variants, for example rocky linux, alma linux, oracle linux.

**Reply by kenneth0595** [reply]
Thank you for your quick response!

I will migrate it to new hypervisor.

**Reply by weizhouapache** [reply, marked as answer]
> Thank you for your quick response!
> 
> I will migrate it to new hypervisor.

good

you can still use centos7 with ACS 4.19.1.3.
the packages are available on http://download.cloudstack.org/centos/7/4.19/
thanks @DaanHoogland for packaging
